torch/_inductor/config.py

```diff
@@ -1153,6 +1153,11 @@ def decide_compile_threads() -> int:
 # decompose some memory bound matmul/bmm to mul
 decompose_mem_bound_mm: bool = False
 
+# Wrap compiled regions in inductor_compiled_code HOP to make them visible to
+# TorchDispatchModes like DebugMode and Selective Activation Checkpointing.
+# This avoids runtime overhead of checking dispatch modes at every call.
```

Comment:
Err, this comment is weird. When this config is on you are checking dispatch mode every call no